src/Examples.Console/Program.cs

```diff
@@ -23,19 +23,17 @@ public class Program
 	{
 		public static async Task Main(string[] args)
 		{
-			await Parser.Default.ParseArguments<DynatraceOptions>(args)
-			.MapResult(
-				(DynatraceOptions options) => DynatraceExporterExample.RunAsync(options.Url, options.ApiToken, options.PushIntervalInSecs, options.DurationInMins, !options.DisableDynatraceMetadataEnrichment),
-				errs => Task.FromResult(0));
-
+			await Parser.Default.ParseArguments<Options>(args)
+				.MapResult(
+				(Options options) => DynatraceExporterExample.RunAsync(options), errs => Task.FromResult(0));
```

Comment:
I know this is just the example, but we could discard (`_`) this parameter and return `1` on failure. :)